### ISSUE TITLE
feature request: color-coded tabs

### ISSUE THREAD

**Issue by kalenedrael** [hultmann/multifox#22]
I'd like a quick visual way of distinguishing what account each tab is using, by coloring each tab according to account.

**Comment by hultmann** [comment]
A way to distinguish between accounts would be cool, but colors may break themes or extensions...

**Comment by Drugoy** [comment]
I agree with you, hultmann. Maybe a good idea would be to add [Login-name] text to the left from the page's title?
2 different tabs would look like this:
[hultmann] #22: feature request: color-coded tabs -Issues - hultmann/multifox - GitHub
[Drugoy] #22: feature request: color-coded tabs -Issues - hultmann/multifox - GitHub

**Comment by hultmann** [comment]
yes, it's a possibility.
